crates/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -805,7 +805,7 @@ pub(super) fn replace(s: &[Series], literal: bool, n: i64) -> PolarsResult<Serie
 #[cfg(feature = "string_from_radix")]
```

Comment:
Can we rename the feature to match the new name?

```suggestion
#[cfg(feature = "string_to_integer")]
```